cmd/ris-mirror/rismirror/rismirror.go

```diff
@@ -35,6 +36,38 @@ func (rism *RISMirror) AddTarget(rtrName string, address net.IP, vrfRD uint64, s
 	r.addVRF(vrfRD, sources)
 }
 
+func (rism *RISMirror) RemoveTarget(rtrName string, vrfRD uint64) {
+	rism.routersMu.Lock()
+	defer rism.routersMu.Unlock()
+
+	if _, exists := rism.routers[rtrName]; !exists {
+		log.Errorf("Router %s does not exist, cannot remove cleanly", rtrName)
+		return
+	}
+	r := rism.routers[rtrName].(*Router)
+	if r.GetVRF(vrfRD) == nil {
+		log.Errorf("VRF %v in Router %v does not exist, cannot remove cleanly", vrfRD)
+		return
+	}
+	r.removeVRF(vrfRD)
+	if len(r.GetVRFs()) < 1 {
```

Comment:
I guess we actually care if we have zero, so lets check `== 0`.

Reply:
Adjusted that.